pkgs/by-name/kl/kleiner-brauhelfer-2/package.nix

```
@@ -0,0 +1,76 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  appimageTools,
+  qt6,
+  makeWrapper,
+}:
+let
+  pname = "kleiner-brauhelfer-2";
+  version = "2.6.2";
+
+  src =
+    fetchurl
+      {
+        x86_64-linux = {
+          url = "https://github.com/kleiner-brauhelfer/kleiner-brauhelfer-2/releases/download/v${version}/kleiner-brauhelfer-2-x86_64.AppImage";
+          hash = "sha256-PsIN7pg5vCrLlMD7Bm1HTmHK+RGyP10ifx842Kzy4H8=";
+        };
+      }
+      .${stdenv.system} or (throw "Unsupported system: ${stdenv.system}");
+
+  meta = with lib; {
+    homepage = "https://kleiner-brauhelfer.de";
+    description = "The 'Kleiner-Brauhelfer' is a helper program for homebrewers to create and manage beer batches.";
+    mainProgram = "kleiner-brauhelfer-2";
+    changelog = "https://github.com/kleiner-brauhelfer/kleiner-brauhelfer-2/releases/tag/v${version}";
+    license = licenses.gpl3;
+    platforms = [ "x86_64-linux" ];
+    maintainers = with maintainers; [ yeoldegrove ];
+  };
+in
+if stdenv.hostPlatform.isLinux then
+  appimageTools.wrapType2 {
+    inherit
+      pname
+      version
+      src
+      meta
+      ;
+
+    nativeBuildInputs = [ makeWrapper ];
+
+    extraPkgs =
+      pkgs: with pkgs; [
+        qt6.qtbase
+        qt6.qtwayland
+        qt6.qtsvg
+        qt6.qttools
+        qt6.qtdeclarative
+      ];
+
+    extraInstallCommands =
+      let
+        appimageContents = appimageTools.extract { inherit pname version src; };
+      in
+      ''
+        # Install XDG Desktop file and its icon
+        install -Dm444 ${appimageContents}/usr/share/applications/${pname}.desktop -t $out/share/applications
```

```suggestion
        install -Dm444 ${appimageContents}/usr/share/applications/kleiner-brauhelfer-2.desktop -t $out/share/applications
```
https://github.com/NixOS/nixpkgs/issues/277994

---

removed `${pname}`  references